diff --git a/jetty-maven-plugin/src/main/java/org/mortbay/jetty/plugin/AbstractJettyMojo.java b/jetty-maven-plugin/src/main/java/org/mortbay/jetty/plugin/AbstractJettyMojo.java
index 1f1f85b..92fbca4 100644
--- org/mortbay/jetty/plugin/AbstractJettyMojo.java
+++ org/mortbay/jetty/plugin/AbstractJettyMojo.java
@@ -29,6 +29,8 @@ import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
 import org.apache.maven.project.MavenProject;
+import org.cipango.plugin.CipangoServer;
+import org.cipango.plugin.CipangoSipAppContext;
 import org.codehaus.plexus.util.FileUtils;
 import org.eclipse.jetty.security.LoginService;
 import org.eclipse.jetty.server.Connector;
@@ -51,7 +53,7 @@ public abstract class AbstractJettyMojo extends AbstractMojo
     /**
      * A wrapper for the Server object
      */
-    protected JettyServer server;
+    protected CipangoServer server;
     
     /**
      * List of connectors to use. If none are configured
@@ -92,7 +94,7 @@ public abstract class AbstractJettyMojo extends AbstractMojo
      * The "virtual" webapp created by the plugin
      * @parameter
      */
-    protected JettyWebAppContext webAppConfig;
+    protected CipangoSipAppContext webAppConfig;
 
 
 
@@ -281,7 +283,7 @@ public abstract class AbstractJettyMojo extends AbstractMojo
             getLog().debug("Starting Jetty Server ...");
 
             printSystemProperties();
-            this.server = new JettyServer();
+            this.server = new CipangoServer();
             setServer(this.server);
 
             //apply any config from a jetty.xml file first which is able to
@@ -379,7 +381,7 @@ public abstract class AbstractJettyMojo extends AbstractMojo
     {
         //As of jetty-7, you must use a <webAppConfig> element
         if (webAppConfig == null)
-            webAppConfig = new JettyWebAppContext();
+            webAppConfig = new CipangoSipAppContext();
         
         //Apply any context xml file to set up the webapp
         //CAUTION: if you've defined a <webAppConfig> element then the
@@ -631,12 +633,12 @@ public abstract class AbstractJettyMojo extends AbstractMojo
     }
 
 
-    public JettyServer getServer ()
+    public CipangoServer getServer ()
     {
         return this.server;
     }
 
-    public void setServer (JettyServer server)
+    public void setServer (CipangoServer server)
     {
         this.server = server;
     }
@@ -663,11 +665,11 @@ public abstract class AbstractJettyMojo extends AbstractMojo
         return this.scannerListeners;
     }
 
-    public JettyWebAppContext getWebAppConfig() {
+    public CipangoSipAppContext getWebAppConfig() {
         return webAppConfig;
     }
 
-    public void setWebAppConfig(JettyWebAppContext webAppConfig) {
+    public void setWebAppConfig(CipangoSipAppContext webAppConfig) {
         this.webAppConfig = webAppConfig;
     }
 
