diff --git a/jetty-osgi/jetty-osgi-boot/src/main/java/org/eclipse/jetty/osgi/boot/JettyBootstrapActivator.java b/jetty-osgi/jetty-osgi-boot/src/main/java/org/eclipse/jetty/osgi/boot/JettyBootstrapActivator.java
index e3c0f90..75ab494 100644
--- org/eclipse/jetty/osgi/boot/JettyBootstrapActivator.java
+++ org/eclipse/jetty/osgi/boot/JettyBootstrapActivator.java
@@ -23,7 +23,7 @@ import org.eclipse.jetty.osgi.boot.internal.webapp.IWebBundleDeployerHelper;
 import org.eclipse.jetty.osgi.boot.internal.webapp.JettyContextHandlerServiceTracker;
 import org.eclipse.jetty.osgi.boot.internal.webapp.WebBundleTrackerCustomizer;
 import org.eclipse.jetty.osgi.boot.utils.internal.PackageAdminServiceTracker;
-import org.eclipse.jetty.server.Server;
+import org.cipango.server.Server;
 import org.eclipse.jetty.server.handler.ContextHandler;
 import org.eclipse.jetty.webapp.WebAppContext;
 import org.osgi.framework.Bundle;
@@ -34,6 +34,8 @@ import org.osgi.framework.FrameworkUtil;
 import org.osgi.framework.ServiceRegistration;
 import org.osgi.util.tracker.BundleTracker;
 
+import org.cipango.sipapp.SipAppContext;
+
 /**
  * Experiment: bootstrap jetty's complete distrib from an OSGi bundle. Progress:
  * <ol>
@@ -212,7 +214,7 @@ public class JettyBootstrapActivator implements BundleActivator
     public static void registerWebapplication(Bundle contributor, String webappFolderPath, String contextPath) throws Exception
     {
     	checkBundleActivated();
-    	WebAppContext contextHandler = new WebAppContext();
+    	SipAppContext contextHandler = new SipAppContext();
         Dictionary dic = new Hashtable();
         dic.put(OSGiWebappConstants.SERVICE_PROP_WAR,webappFolderPath);
         dic.put(OSGiWebappConstants.SERVICE_PROP_CONTEXT_PATH,contextPath);
@@ -242,7 +244,7 @@ public class JettyBootstrapActivator implements BundleActivator
     public static void registerWebapplication(Bundle contributor, String webappFolderPath, String contextPath, Dictionary<String, String> dic) throws Exception
     {
     	checkBundleActivated();
-        WebAppContext contextHandler = new WebAppContext();
+        SipAppContext contextHandler = new SipAppContext();
         dic.put(OSGiWebappConstants.SERVICE_PROP_WAR,webappFolderPath);
         dic.put(OSGiWebappConstants.SERVICE_PROP_CONTEXT_PATH,contextPath);
         contributor.getBundleContext().registerService(ContextHandler.class.getName(),contextHandler,dic);
