diff --git a/jetty-maven-plugin/src/main/java/org/mortbay/jetty/plugin/MavenWebInfConfiguration.java b/jetty-maven-plugin/src/main/java/org/mortbay/jetty/plugin/MavenWebInfConfiguration.java
index 4c7547f..ec932cd 100644
--- org/mortbay/jetty/plugin/MavenWebInfConfiguration.java
+++ org/mortbay/jetty/plugin/MavenWebInfConfiguration.java
@@ -32,6 +32,8 @@ import org.eclipse.jetty.webapp.WebAppClassLoader;
 import org.eclipse.jetty.webapp.WebAppContext;
 import org.eclipse.jetty.webapp.WebInfConfiguration;
 
+import org.cipango.plugin.CipangoSipAppContext;
+
 public class MavenWebInfConfiguration extends WebInfConfiguration
 {
     private static final Logger LOG = Log.getLogger(MavenWebInfConfiguration.class);
@@ -42,7 +44,7 @@ public class MavenWebInfConfiguration extends WebInfConfiguration
     
     public void configure(WebAppContext context) throws Exception
     {
-        JettyWebAppContext jwac = (JettyWebAppContext)context;
+        CipangoSipAppContext jwac = (CipangoSipAppContext)context;
         if (jwac.getClassPathFiles() != null)
         {
             if (LOG.isDebugEnabled()) LOG.debug("Setting up classpath ...");
@@ -78,7 +80,7 @@ public class MavenWebInfConfiguration extends WebInfConfiguration
         super.preConfigure(context);
 
         _originalResourceBase = context.getBaseResource();
-        JettyWebAppContext jwac = (JettyWebAppContext)context;
+        CipangoSipAppContext jwac = (CipangoSipAppContext)context;
 
         //Add in any overlaid wars as base resources
         if (jwac.getOverlays() != null && !jwac.getOverlays().isEmpty())
@@ -142,7 +144,7 @@ public class MavenWebInfConfiguration extends WebInfConfiguration
 
     public void deconfigure(WebAppContext context) throws Exception
     {
-        JettyWebAppContext jwac = (JettyWebAppContext)context;
+        CipangoSipAppContext jwac = (CipangoSipAppContext)context;
         
         //remove the unpacked wars
         if (_unpackedOverlays != null && _unpackedOverlays.length>0)
@@ -177,7 +179,7 @@ public class MavenWebInfConfiguration extends WebInfConfiguration
     throws Exception
     {
         List<Resource> list = new ArrayList<Resource>();
-        JettyWebAppContext jwac = (JettyWebAppContext)context;
+        CipangoSipAppContext jwac = (CipangoSipAppContext)context;
         if (jwac.getClassPathFiles() != null)
         {
             for (File f: jwac.getClassPathFiles())
